src/components/customButton/CustomButton.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Button, StyleSheet, View } from 'react-native';
+
+import { Colors } from 'utils/constants/Colors';
+
+interface Props {
+    title: string;
+    onPress: () => void;
+    disabled?: boolean;
+}
+
+export const CustomButton = ({ title, onPress, disabled }: Props) => (
```

Comment:
`{ title, onPress, disabled }` to `props`

Reply:
fixed